subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/GraphiteSender.kt

```diff
@@ -0,0 +1,49 @@
+package com.avito.android.graphite
+
+import java.io.DataOutputStream
+import java.net.Socket
+
+
+interface GraphiteSender {
+
+    /**
+     * Incubating
+     */
+    fun send(metric: GraphiteMetric)
+
+    class Impl(
+        private val config: GraphiteConfig,
+        private val logger: (String, Throwable?) -> Unit
+    ) : GraphiteSender {
+
+        override fun send(metric: GraphiteMetric) {
+            val metricName = if (config.namespace.isEmpty()) {
+                metric.path
+            } else {
+                config.namespace.removeSuffix(".") + "." + metric.path
+            }
+
+            send(metricName, metric.value, metric.timestamp)
+        }
+
+        private fun send(path: String, value: String, timestamp: Long) {
+            // https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol
+            val message = "$path $value $timestamp\n"
+
+            logger.invoke("graphite: $message", null)
+
+            if (!config.isEnabled) return
```

Comment:
It's look ugly. I can see in log send info but send woun't be happen
```suggestion
             if (!config.isEnabled) {
              // ignore info
             } else {
             // send info
             }
```

Reply:
fixed. I've added "debug" mode in config